### DISCUSSION TITLE
Question (or feature req): mount to different locations

### DISCUSSION THREAD

**Discussion by dsmith4-godaddy** [lima-vm/lima#1663] [Ideas]
I would like to be able to mount directories from the host, to a different location inside the guest. 

Example: on my host Mac, my home dir is /Users/dsmith , but I'd like to mount that directory under the guest's /home/dsmith.linux (so that things like `cd` work similarly on both). I suppose I could edit the guest's home dir, but that would only address this specific case, and this feels like something more useful generally.

I'm not sure what the least-ambiguous yaml syntax would be for this, as the current "location" works great - when the location on both host and guest are the same. You'd probably have to have either a "guestLocation" (and the parent is the host) or a "hostLocation" (and the parent is the guest). Or in theory both, making the location itself redundant, but that could get a bit hairy to parse, and you'd probably need to keep support for location around anyway to avoid breaking backwards compatibility.

**Comment by dsmith4-godaddy** [comment]
Following up to myself: After a few minutes of digging through default.yaml I found this already exists, it just wasn't in the docs:

```
mounts:
- location: "/Users/dsmith"
  mountPoint: "/home/dsmith.linux"
  writable: true
```

In the copious free time I'll see if I can find a way to update the mounts doc, to call this out without making the whole page unnecessarily bigger/more complicated.

**Reply by afbjorklund** [reply]
Mounting a remote location like your home directory as the instance user's home dir is not supported, and you risk having conflicts between various configuration files between local and remote.

But the main issue is that container storage cannot be remote. If you absolutely need such a feature, then mounting a few subdirectories from your home would be better.
